packages/ds-medicare-gov/src/styles/components/_Navbar.scss

```diff
@@ -1,42 +1,45 @@
 .m-c-navbar {
-  @extend .ds-u-display--flex;
-  @extend .ds-u-justify-content--between;
-  @extend .ds-u-align-items--start;
-  @extend .ds-u-sm-align-items--center;
-  @extend .ds-u-fill--primary;
-  @extend .ds-u-margin-bottom--2;
-  @extend .ds-u-sm-margin-bottom--0;
-  @extend .ds-u-padding-x--0;
-  @extend .ds-u-sm-padding-x--7;
+  display: flex;
+  justify-content: flex-between;
+  align-items: flex-start;
   flex-direction: column;
+  background-color: $color-primary;
+  margin-bottom: $spacer-2;
+  padding-left: 0;
+  padding-right: 0;
   position: relative;
 
   @media screen and (min-width: $width-sm) {
+    align-items: center;
     flex-direction: row;
     position: static;
+    margin-bottom: 0;
+    padding-left: $spacer-7;
+    padding-right: $spacer-7;
   }
 
   &.m-c-navbar--inverse {
-    @extend .ds-u-fill--white;
+    background-color: $color-white;
   }
 }
 
 .m-c-navbar__section {
   @extend .ds-l-col--12;
   @extend .ds-l-sm-col--6;
-  @extend .ds-u-padding-x--0;
-  @extend .ds-u-display--flex;
-  @extend .ds-u-align-items--start;
-  @extend .ds-u-sm-align-items--center;
+  padding-left: 0 !important; // Has to override the column class styles
+  padding-right: 0 !important; // Has to override the column class styles
```

Comment:
I bounded my changes to only the utility classes, which means I didn't touch the use of `ds-l-*` classes here. In order to override the padding from those, I had to use important. I commented everywhere I did that, though.

Reply:
It could totally use another pass of simplifying—maybe when we start mapping components to design token themes?

Reply:
I'm not sure. Off the top of my head, I don't know of a better way to do that in Sass